### DISCUSSION TITLE
Copy file path at specific line number

### DISCUSSION THREAD

**Discussion by deividaspetraitis** [community/community#7601] [Mobile]
Title says everything. It is more convenient copy file path with specified line number than copy file path and write number manually.

Edit:

I am on iOS

**Comment by brianlovin** [comment]
@deividaspetraitis agree — are you on iOS or Android?

**Reply by deividaspetraitis** [reply]
Sorry for not mentioning in original message, I am on iOS.

**Reply by brianlovin** [reply]
Thanks — opening an issue for the team now so that the share button in the navigation bar will append line number query parameters if lines are selected.

**Reply by deividaspetraitis** [reply]
Sounds great, thank you!
